src/indigo/less/sidebar.less

```diff
@@ -51,8 +51,9 @@
           position: absolute;
           top: 24px;
           right: 20px;
-          @media all and (max-height: 770px) {
```

Comment:
Nemělo tady být původně už max-width?

Reply:
aha, tak to tam vrátím. Já to původně urpavil omylem, že jsme si nevšiml že to je height, a pak myslel že to je bug tak jsem to nechal. Tak to vrátím.